server/chat-plugins/mafia.ts

```diff
@@ -2656,15 +2813,19 @@ export const commands: ChatCommands = {
 			if (game.hostid !== user.id && !game.cohosts.includes(user.id)) return this.errorReply(`Only the host can view roles.`);
 			if (!game.started) return this.errorReply(`The game has not started.`);
 			const players = [...Object.values(game.playerTable), ...Object.values(game.dead)];
-			this.sendReplyBox(players.map(p => `${p.safeName}: ${p.role ? (p.role.alignment === 'solo' ? 'Solo ' : '') + p.role.safeName : 'No role'}`).join('<br/>'));
+			this.sendReplyBox(players.map(p => {
+				return `${p.safeName}: ${p.role ? (p.role.alignment === 'solo' ? 'Solo ' : '') + p.role.safeName : 'No role'}`;
+			}).join('<br/>'));
```

Comment:
Please please please don't add the `return` keyword to anything that didn't previously have it.

You can _always_ do:

```suggestion
			this.sendReplyBox(players.map(p => (
				`${p.safeName}: ${p.role ? (p.role.alignment === 'solo' ? 'Solo ' : '') + p.role.safeName : 'No role'}`
			)).join('<br/>'));
```

But what I'd actually suggest here is

```suggestion
			this.sendReplyBox(players.map(
				p => `${p.safeName}: ${p.role ? (p.role.alignment === 'solo' ? 'Solo ' : '') + p.role.safeName : 'No role'}`;
			).join('<br/>'));
```